package.json

```diff
@@ -29,7 +29,6 @@
     "example": "examples"
   },
   "scripts": {
-    "postinstall": "[ \"$NODE_ENV\" != production ] && npm ln ../jsdom && npm ln ../buble-tape",
```

Comment:
@brandondees @VicenteRD did you two experience errors running `npm run browse` after installing package? @mrbernnz experienced this and i believe i did once as well. An error on `postinstall`. @mrbernnz will yarn help with this as well? Or different issue?